modules/EnsEMBL/Web/Component/Gene/Interaction.pm

```diff
@@ -0,0 +1,156 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package EnsEMBL::Web::Component::Gene::Interaction;
+use strict;
+
+use URI::Escape qw(uri_escape);
+use HTML::Entities qw(encode_entities);
+use JSON qw(from_json to_json encode_json decode_json);
+
+use base qw(EnsEMBL::Web::Component::Gene);
+use Data::Dumper;
+$Data::Dumper::Maxdepth = 4; $Data::Dumper::Sortkeys = 1;
+
```

Comment:
Presumably for debugging? Best to delete these, as (IIRC) the settings will affect others' sandboxes.